.github/workflows/test.yml

```diff
@@ -20,6 +20,7 @@ jobs:
 
     - name: Install dependencies
       run: |
+        sudo apt update -y
```

Comment:
I noticed that ffmpeg wasn't installing the afternoon without updating apt first.